src/main/java/com/openlattice/chronicle/api/ChronicleApi.java

```diff
@@ -160,6 +162,16 @@ void sendMessages(
             @Body List<MessageDetails> messageDetailsList
     );
 
+    /**
+     * Update staus for messages sent to partipants.
+     * @param organizationId - Id of the organization to which study belongs
+     */
+    @POST( BASE + ORGANIZATION_ID_PATH + MESSAGE_PATH + STATUS_PATH)
+    void updateMessageStatus(
+            @Path( ORGANIZATION_ID ) UUID organizationId,
+            HttpServletRequest request
```

Comment:
`HttpServletRequest request` is a Controller thing and does not belong in the API.

Reply:
I think this is what you're looking for:

```
@POST( BASE + ORGANIZATION_ID_PATH + MESSAGE_PATH + STATUS_PATH)
void updateMessageStatus(
  @Path( ORGANIZATION_ID ) UUID organizationId,
  @Query( MESSAGE_SID ) String messageSid,
  @Query( MESSAGE_STATUS ) String MessageStatus
)
```